.circleci/config.yml

```diff
@@ -0,0 +1,40 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine.
+# See: https://circleci.com/docs/configuration-reference
+version: 2.1
+
+# Define a job to be invoked later in a workflow.
+# See: https://circleci.com/docs/jobs-steps/#jobs-overview & https://circleci.com/docs/configuration-reference/#jobs
+jobs:
+  build:
+    # Specify the execution environment. You can specify an image from Docker Hub or use one of our convenience images from CircleCI's Developer Hub.
+    # See: https://circleci.com/docs/executor-intro/ & https://circleci.com/docs/configuration-reference/#executor-job
+    docker:
+      # Specify the version you desire here
+      # See: https://circleci.com/developer/images/image/cimg/base
+      - image: circleci/ruby:2.3.8-node-browsers
+    environment:
+      RAILS_VERSION: "3.2.22.5"
+    # Add steps to the job
+    # See: https://circleci.com/docs/jobs-steps/#steps-overview & https://circleci.com/docs/configuration-reference/#steps
+    steps:
+      # Checkout the code as the first step.
+      - checkout
+      - run:
+          name: "Say hello"
+          command: "echo Hello, World!"
+      - run:
```

Comment:
Minor, but you could have use the [`ruby` ORB](https://circleci.com/developer/orbs/orb/circleci/ruby) as done on [this gem](https://github.com/bodyshopbidsdotcom/pdf-reader/blob/main-snapsheet/.circleci/config.yml#L14-L15). The `ruby/rspec-test` command also supports a [`tag` attribute](https://circleci.com/developer/orbs/orb/circleci/ruby#commands-rspec-test)